src/ray/raylet/scheduling/cluster_resource_manager.cc

```diff
@@ -59,8 +59,6 @@ void ClusterResourceManager::AddOrUpdateNode(
 
 void ClusterResourceManager::AddOrUpdateNode(scheduling::NodeID node_id,
                                              const NodeResources &node_resources) {
-  RAY_LOG(DEBUG) << "Update node info, node_id: " << node_id.ToInt()
```

Comment:
Why remove?

Reply:
This seems useless and too verbose. I can bring it back if you think it is necessary